templates/flavors/base/cluster-template.yaml

```diff
@@ -74,6 +74,9 @@ spec:
             mountPath: /etc/kubernetes/azure.json
             name: cloud-config
             readOnly: true
+      etcd:
+        local:
+          dataDir: "/var/lib/etcddisk/etcd"
```

Comment:
This is to address kubernetes/kubeadm#2127. The two alternatives were:
1) Add a preKubeadmCommand to ` rm -rf /var/lib/etcd`
2) Add a preKubeadmCommand to create a symlink from `/var/lib/etcd` to `/var/lib/etcddisk/etcd`
3) Change the etcd data dir to `/var/lib/etcddisk/etcd` (default is /var/lib/etcd)
I decided to go with 3) because it doesn't pollute preKubeadmCommand with infra required stuff (it should be reserved for the user).